src/Service/Route53/Makefile

```diff
@@ -0,0 +1,12 @@
+.EXPORT_ALL_VARIABLES:
+
+initialize: start-docker
+start-docker:
+	echo "Noop"
```

Comment:
LocalStack provides a service for Route53

You can take inspiration from `SSM` (see `Makefile`, `src/Service/Ssm/Makefile` and `SsmClientTest::getClient`)

Reply:
Nice, I'll take a look